LastHitMarker/Program.cs

```diff
@@ -68,19 +75,34 @@ private static void Game_OnUpdate(EventArgs args)
         private static void Drawing_OnDraw(EventArgs args)
         {
             if (!Game.IsInGame || Game.GameTime > 1800)
+            {
+                if (CreepsDictionary.Count > 0) CreepsDictionary.Clear();
+                if (CreepsTeamDictionary.Count > 0) CreepsTeamDictionary.Clear();
                 return;
+            }
+
+            if (!_textureLoaded)
+            {
+                _textureActiveCoin = Drawing.GetTexture("materials/ensage_ui/other/active_coin.vmat");
+                _textureActiveDeny = Drawing.GetTexture("materials/ensage_ui/other/active_deny.vmat");
+                _texturePassiveCoin = Drawing.GetTexture("materials/ensage_ui/other/passive_coin.vmat");
+                _texturePassiveDeny = Drawing.GetTexture("materials/ensage_ui/other/passive_deny.vmat");
+                _textureLoaded = true;
+            }
+
+            foreach (var keyPair in CreepsDictionary)
```

Comment:
What is faster than looping through all of the creeps ? It's looping through the known list of creep :P